zipline/pipeline/data/dataset.py

```diff
@@ -251,7 +281,14 @@ def fx(self, currency):
             into ``currency``.
         """
         conversion = self._currency_conversion
-        if conversion is not None and conversion.currency == currency:
+        currency_aware = self._currency_aware
+
+        if not currency_aware:
```

Comment:
This can just be `if not self._currency_aware`. The reason I'm binding `conversion` as a local variable here is because the if-statement below exceeds 80 chars if we use `self._currency_conversion` twice.